build/buildBuildImages.sh

```diff
@@ -146,7 +146,7 @@ function buildGitHubActionsImage() {
 	
 	echo
 	echo "-------------Creating build image for GitHub Actions-------------------"
-	builtImageName="$ACR_BUILD_GITHUB_ACTIONS_IMAGE_NAME"
+	local builtImageName="$ACR_BUILD_GITHUB_ACTIONS_IMAGE_NAME"
```

Comment:
wondering what `local` here used for

Reply:
oh never mind, got it

Reply:
This is so that scope of this variable is limited to this function only